pom.xml

```diff
@@ -116,12 +128,6 @@
       <groupId>commons-beanutils</groupId>
       <artifactId>commons-beanutils</artifactId>
       <version>1.10.0</version>
-      <exclusions>
```

Comment:
we are already using commons-logging, so this exclusion was old and wrong.  As noted nothing actually checks for that, it requires humans to do so.

Reply:
I don't understand this comment. It wasn't being excluded because it wasn't being used. It was being excluded so we could specify a managed version, and it was excluded here so that the managed version did not conflict with this version in the dependencyConvergence rule.

Yes, it requires humans to check... but that's the point. A human should triage dependencyConvergence issues.